git_greppers/git_grepper.sh

```
@@ -0,0 +1,161 @@
+fi
+printf "${CYAN}Output will be saved to: %s/${NC}\n" "$OUTPUT_DIR"
+
+# Create output directory if it doesn't exist
+mkdir -p "$OUTPUT_DIR"
+debug 1 "Created output directory: $OUTPUT_DIR"
+
+# Create log file
+LOG_FILE=$(create_log_file "$OUTPUT_DIR" "$PRIMARY_BRANCH" "$REPO_FOLDER" "$MAX_DIFF_LINES")
+
+# Get list of commits sorted by date (oldest first)
+COMMITS=$(git log --after="2 months ago" --format="%H" $PRIMARY_BRANCH)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)